src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  health = 100;
+
+  constructor(name) {
+    this.name = name;
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  constructor(name) {
+    super(name);
+
+    Animal.alive.push(this);
+  }
+
+  hide() {
```

Comment:
The hide method in the Herbivore class should only set the hidden property to true, as per the checklist. It should not toggle the value.